install_pim/manual/system_requirements/manual_system_installation_debian10.rst

```diff
@@ -12,61 +12,49 @@ application to match your local installation.
 System installation
 -------------------
 
-MySQL 5.7
+MySQL 8.0
 *********
 
-The easiest way to install MySQL 5.7 is to use the official vendor package.
+The easiest way to install MySQL 8.0 is to use the official vendor package.
 
 First, start by installing the `MySQL APT repository <https://dev.mysql.com/doc/mysql-apt-repo-quick-guide/en/#apt-repo-setup>`_.
 It's a tool that aims to ease the installation and update of MySQL products.
-During the installation of this tool, one will be asked to choose the versions of the MySQL server to install. MySQL 5.7 has to be chosen.
+During the installation of this tool, one will be asked to choose the versions of the MySQL server to install. MySQL 8.0 has to be chosen.
 It's also recommended to disable all non desired tools, such as MySQL Workbench or MySQL Router.
 
 .. code-block:: bash
 
-    # apt install lsb-release apt-transport-https ca-certificates
-    # wget -O mysql-apt-config.deb https://dev.mysql.com/get/mysql-apt-config_0.8.7-1_all.deb
+    # apt install lsb-release apt-transport-https ca-certificates wget gnupg
+    # wget -O mysql-apt-config.deb https://dev.mysql.com/get/mysql-apt-config_0.8.13-1_all.deb
     # dpkg -i mysql-apt-config.deb
-    # apt install dirmngr
-    # apt-key adv --keyserver keys.gnupg.net --recv-keys 8C718D3B5072E1F5
 
 Now is the time to install what has been configured in the step before:
 
 .. code-block:: bash
 
-    # apt update
-    # apt install mysql-server
+    # apt update && apt install mysql-server
 
-PHP 7.2
-*******
-
-The easiest way to install PHP 7.2 is to use `Ondrej Sury <https://deb.sury.org/>`_ packages.
-
-First, install the `repository <https://packages.sury.org/php/README.txt>`_:
-
-.. code-block:: bash
+When installing MySQL 8.0, you'll have to choose the authentication method. Please select *Use Legacy Authentication Method* as the *Strong Password Encryption* is not yet supported by Akeneo PIM.
 
-    # apt install apt-transport-https ca-certificates
-    # wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg
-    # sh -c 'echo "deb https://packages.sury.org/php/ stretch main" > /etc/apt/sources.list.d/php.list'
-    # apt update
+PHP 7.3
+*******
 
-Then, install PHP and the required extensions:
+Install PHP and the required extensions:
 
 .. code-block:: bash
 
-    # apt install php7.2-apcu php7.2-bcmath php7.2-cli php7.2-curl php7.2-fpm php7.2-gd php7.2-intl php7.2-mysql php7.2-xml php7.2-zip
+    # apt install php7.3-cli php7.3-apcu php7.3-bcmath php7.3-curl php7.3-fpm php7.3-gd php7.3-intl php7.3-mysql php7.3-xml php7.3-zip php7.3-zip php7.3-mbstring
 
 For Enterprise Edition, please also install:
```

Comment:
I think that now we check every system components from the CE edition, even the one currently only used in EE. This simplifies our work on this point and reduces tech stack differences between both editions.
So here, we must not separate PHP requirements. It may make the CE installation fail ;)